--- captive-1.1.5/src/client/lufs/mount.captive.in.orig	2003-12-02 18:23:19.000000000 +0100
+++ captive-1.1.5/src/client/lufs/mount.captive.in	2005-03-26 23:19:21.163928008 +0100
@@ -28,8 +28,7 @@
 my $vardir='@localstatedir@/lib/captive';
 $vardir=~s#\$\Q{prefix}\E#'@prefix@';#ge;
 $vardir="/var/lib/captive" if $vardir=~/^@/;
-my $lufsd_bin='@PATH_LUFSD@';
-$lufsd_bin="/usr/bin/lufsd" if $lufsd_bin=~/^@/;
+my $lufsd_bin="/usr/bin/lufis";
 my $captive_sandbox_server_bin='@sbindir@/captive-sandbox-server';
 $captive_sandbox_server_bin=~s#\$\Q{exec_prefix}\E#'@exec_prefix@';#ge;
 $captive_sandbox_server_bin=~s#\$\Q{prefix}\E#'@prefix@';#ge;
@@ -128,8 +127,10 @@
 
 warn "$0: '-n' not supported - ignored" if $nomtab;
 
+my $fuseopts="-oallow_other,fsname=$image";
+
 # Use "'$oo'" to permit another expansion by bash(1) during lufsmnt(8) execution.
-my @argv=($lufsd_bin,"none",$dir,"-o",$oo);
+my @argv=($lufsd_bin,$oo,$dir,"-s",$fuseopts);
 
 print STDERR "$0: @argv\n" if $verbose;
 exec $lufsd_bin @argv;
